app/src/main/java/org/mozilla/focus/browser/ErrorPage.java

```diff
@@ -6,91 +6,21 @@
 
 import android.content.Context;
 import android.content.res.Resources;
+import android.support.annotation.NonNull;
 import android.support.v4.util.ArrayMap;
-import android.support.v4.util.Pair;
 
-import android.util.SparseArray;
-import android.webkit.WebViewClient;
 import org.mozilla.focus.R;
 import org.mozilla.focus.utils.HtmlLoader;
 
 import java.util.Map;
 
-public class ErrorPage {
-
-    private static final SparseArray<Pair<Integer, Integer>> errorDescriptionMap;
-
-    static {
-        errorDescriptionMap = new SparseArray<>();
-
-        // Chromium's mapping (internal error code, to Android WebView error code) is described at:
-        // https://chromium.googlesource.com/chromium/src.git/+/master/android_webview/java/src/org/chromium/android_webview/ErrorCodeConversionHelper.java
-
-        errorDescriptionMap.put(WebViewClient.ERROR_UNKNOWN,
-                new Pair<>(R.string.error_connectionfailure_title, R.string.error_connectionfailure_message));
```

Comment:
Should we delete all these strings too? I assume they're unused.

Reply:
Note that components do not contain any translations yet. We are waiting for the new L10N tooling to. be ready. You may want to keep the strings to keep the translations for now - I think?

Reply:
So does that mean we can't land this because the current strings are being pulled from components, not strings.xml?

Reply:
You can land this. We kept the string names the same so that the strings from Focus / FFFFTV will override them.

Reply:
But I assume that if you remove the Strings here then the next string export/import will remove all the translations!